### PR TITLE
New package: wayclip-0.3

### PR DESCRIPTION
**Pull request by nvidiaLinuxUser**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by nvidiaLinuxUser** [comment]
Closes #20077

**Comment by dkwo** [comment]
It builds fine on x86_64-musl.
Should it depend on wlroots?

**Comment by nvidiaLinuxUser** [comment]
> It builds fine on x86_64-musl.

Nice :)

> Should it depend on wlroots?

I think no

**Comment by nvidiaLinuxUser** [comment]
I think I made a mistake somewhere

File: srcpkgs/wayclip/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'wayclip'
+pkgname=wayclip
+version=0.2
+revision=1
+build_style=gnu-makefile
+hostmakedepends="wayland-devel"
```
Comment (paper42) [inline srcpkgs/wayclip/template:6]:
```suggestion
makedepends="wayland-devel"
```

Comment (nvidiaLinuxUser) [reply srcpkgs/wayclip/template:6]: Thanks

Comment (classabbyamp) [reply srcpkgs/wayclip/template]: it needs to be in both `hostmakedepends` and `makedepends` (it runs `wayland-scanner`)

Comment (nvidiaLinuxUser) [reply srcpkgs/wayclip/template]: @classabbyamp, got it, thanks :D

**Comment by nvidiaLinuxUser** [comment]
Will it be merged soon? :\

**Review by jcgruenhage — APPROVED** [review]
Looked at upstream, looked at this template. Looks good to me. Good to merge IMO.

**Comment by jcgruenhage** [comment]
@nvidiaLinuxUser can you rebase this?

**Comment by nvidiaLinuxUser** [comment]
@jcgruenhage I don't know much about GitHub ((
It would be nice if someone could merge this instead of me or tell me how to do it

**Comment by jcgruenhage** [comment]
@nvidiaLinuxUser Rebase is something you do locally on your checked out branch: `git fetch origin && git rebase origin/master` should do it. Your branch has split off from master at some point, and rebasing on master basically means that the changes to master since your branch has split off are taken and your changes are applied on top of the new state of the master branch.

If you have trouble with git, https://github.com/git-guides is a good resource, and once you're comfortable with those basics, https://git-rebase.io/ would be a good next step. If you want someone else to take over for this package, until you're more comfortable with git, I'd be happy to do so.

**Comment by jcgruenhage** [comment]
Ah, I see that this is on a `patch-1` branch, so you worked on this completely in the web interface? In that case it might be a bit more complicated.

**Comment by nvidiaLinuxUser** [comment]
> @nvidiaLinuxUser Rebase is something you do locally on your checked out branch: `git fetch origin && git rebase origin/master` should do it. Your branch has split off from master at some point, and rebasing on master basically means that the changes to master since your branch has split off are taken and your changes are applied on top of the new state of the master branch.
> 
> If you have trouble with git, https://github.com/git-guides is a good resource, and once you're comfortable with those basics, https://git-rebase.io/ would be a good next step.

Thanks for the information, I will try to learn it :)

> If you want someone else to take over for this package, until you're more comfortable with git, I'd be happy to do so.
That would be nice

**Comment by nvidiaLinuxUser** [comment]
> Ah, I see that this is on a `patch-1` branch, so you worked on this completely in the web interface? In that case it might be a bit more complicated.

Almost, I did it through GitHub CLI

**Pull request closed without merging**